documentation/source/install_devel.rst

```diff
@@ -23,7 +23,7 @@ than the stable version:
    * pytest
 
 * Python 3.5+
-* Python-dev packages
```

Comment:
I think this might be referring to `apt install python-dev` or similar, but I'm not sure

Reply:
We later list the concrete package names, so I figured this change would be understandable for readers.